server/tests/auth/test_command_policy.py

```
@@ -0,0 +1,753 @@
+
+
+@pytest.fixture()
+def policy(monkeypatch):
+    """Replace ``_get_cached`` with a deterministic, mutable stub."""
+    state = {
+        "allow": [],
+        "deny": [],
+        "states": ListStates(allowlist_enabled=False, denylist_enabled=False),
+    }
+
+    def fake_get_cached(_org_id):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `fake_get_cached`

(ANN202)